airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -80,7 +80,7 @@ def _generate_queued_event_where_clause(
 
 @assets_router.get(
     "/assets",
-    responses=create_openapi_http_exception_doc([401, 403, 404]),
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
```

Comment:
Also removing the 401 and 403 as it comes from the top level router now

Reply:
Its already approved. Can you merge it?